docs/providers/identity-server4.md

```diff
@@ -48,11 +47,10 @@ providers: [
   IdentityServer4Provider({
     id: "demo-identity-server",
     name: "Demo IdentityServer4",
-    scope: "openid profile email api offline_access",
-    domain:  "demo.identityserver.io",
+    authorization: { params: { scope: "openid profile email api offline_access" } },
+    issuer:  "https://demo.identityserver.io/",
```

Comment:
I am fairly sure I tested this locally, and I don't think I needed any of these changes. But I will check again

Reply:
You were right here! :+1: